src/main/java/org/frankframework/frankdoc/model/ElementType.java

```diff
@@ -104,14 +104,14 @@ private static String parseDefaultElementTag(FrankClass clazz, FrankClassReposit
 		if(StringUtils.isBlank(value)) {
 			// null means the JavaDoc tag was not present - then nothing to do.
 			if(value != null) {
-				log.warn("JavaDoc tag {} of interface [{}] should have a parameter", JAVADOC_DEFAULT_CLASSNAME, clazz.getName());
+				log.error("JavaDoc tag {} of interface [{}] should have a parameter", JAVADOC_DEFAULT_CLASSNAME, clazz.getName());
```

Comment:
Dit lijkt me geen reden om de build te laten falen

Reply:
Dit betekent dat de contributor een ff.defaultElement heeft toegevoegd zonder daarbij een class name te noemen. Dit is duidelijk fout en het is gemakkelijk te fixen. Als een contributor dit laat staan, dan maakt hij de code onhelder; hij kan het tag dan net zo goed weghalen. Als ik hier weer een warning van maak, dan missen we volgens mij een kans. Wil je dat echt?

Reply:
ok, ik snap je punt. Laat het maar error hier